### ISSUE TITLE
Stats (Redraws, Completed%, TimeSpentPlaying) - Unity & Frontend

### ISSUE THREAD

**Issue by dobrodejm** [Group-9-Semester-4/ninja-game#177]
TLDR - Unity: Refer to `FinishGameParam` in the backend and send the correct data to the `api/game/finish/` endpoint.

Include Card Set Completion (how many cards were completed from a set during one game) and time spent playing game. Both are a part of FinishGameParam, and should be tracked. This means, both of these variables will be sent along with the FinishGameParam. Check out `insertGameStatistics()` function inside `StatisticsService.java`
The database and backend is finished for all the statistics we want to track. 

Admin panel: Missing Card Set Completed % and Time Spent Playing graphs/views.
